cala-ledger/src/tx_template/mod.rs

```diff
@@ -0,0 +1,47 @@
+mod cel_context;
+mod entity;
+mod repo;
+
+pub mod error;
+
+use sqlx::PgPool;
+use tracing::instrument;
+
+use crate::{entity::*, outbox::*};
+
+pub use entity::*;
+use error::*;
+use repo::*;
+
+#[derive(Clone)]
+pub struct TxTemplates {
+    repo: TxTemplateRepo,
+    _outbox: Outbox,
+    pool: PgPool,
+}
+
+impl TxTemplates {
+    pub fn new(pool: &PgPool, outbox: Outbox) -> Self {
+        Self {
+            repo: TxTemplateRepo::new(pool),
+            _outbox: outbox,
+            pool: pool.clone(),
+        }
+    }
+
+    #[instrument(name = "cala_ledger.accounts.create", skip(self))]
+    pub async fn create(
+        &self,
+        new_tx_template: NewTxTemplate,
+    ) -> Result<TxTemplate, TxTemplateError> {
+        let mut tx = self.pool.begin().await?;
+        let EntityUpdate {
+            entity: tx_template,
+            ..
+        } = self.repo.create_in_tx(&mut tx, new_tx_template).await?;
+        // self.outbox
```

Comment:
why is this commented?

Reply:
commented to make the compiler pass checks. Will work on adding the outbox in the next pr